docs/configuration.md

```diff
@@ -39,7 +39,7 @@ Iceberg tables support table properties to configure table behavior, like the de
 | read.split.metadata-target-size   | 33554432 (32 MB)   | Target size when combining metadata input splits       |
 | read.split.planning-lookback      | 10                 | Number of bins to consider when combining input splits |
 | read.split.open-file-cost         | 4194304 (4 MB)     | The estimated cost to open a file, used as a minimum weight when combining splits. |
-| read.parquet.vectorization.enabled| false              | Enables parquet vectorized reads                       |
+| read.parquet.vectorization.enabled| true               | Enables parquet vectorized reads                       |
```

Comment:
Wonder whether we should update the Description now.

"Controls whether Parquet vectorized reads are used." ?

Reply:
Currently there're two flavors of descriptions for configuration options suffixed by `.enabled`, I'll update the descriptions for vectorized reads in this patch.

Reply:
> two flavors of descriptions for configuration options suffixed by `.enabled`,

I think the difference may come from the distinction whether the option is on or off by default.

For something that's off by default, the wording "Enables ..." is appropriate.
For something that's on by default, it's no longer as nice IMO.